src/output.cpp

```diff
@@ -1756,28 +1756,37 @@ get_hp_bar( const int cur_hp, const int max_hp, const bool is_mon )
            ( ratio >  0.0 )            ? strings[10] : strings[11];
 }
 
-std::pair<std::string, nc_color> const &get_light_level( const float light )
+std::pair<std::string, nc_color> const get_light_level( const float light )
```

Comment:
Please remove the first const here, the function doesn't need to return a const value.